### PR TITLE
[Improve][Connector-V2][StarRocks]Add StarRocks connector option rules

### PR DESCRIPTION
**Pull request by TaoZex**
## Purpose of this pull request

Add StarRocks connector option rules

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)

### CODE REVIEW

File: seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/config/SinkConfig.java [hunk 1]
```diff
@@ -34,22 +36,75 @@
...
+
+    public static final Option<String> STARROCKS_SINK_CONFIG_PREFIX = Options.key("sink.properties.")
+            .stringType()
+            .noDefaultValue()
+            .withDescription("The parameter of the stream load data_desc. " +
+                    "The way to specify the parameter is to add the prefix `sink.properties.` to the original stream load parameter name ");
+
+    public static final Option<Integer> BATCH_MAX_SIZE = Options.key("batch_max_rows")
+            .intType()
+            .noDefaultValue()
```
Comment (EricJoy2048) [inline seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/config/SinkConfig.java]: Check document again?
image

File: seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/sink/StarRocksSink.java [hunk 2]
```diff
@@ -53,7 +53,7 @@ public String getPluginName() {
     @Override
     public void prepare(Config pluginConfig) throws PrepareFailException {
         this.pluginConfig = pluginConfig;
-        CheckResult result = CheckConfigUtil.checkAllExists(pluginConfig, NODE_URLS, DATABASE, TABLE, USERNAME, PASSWORD);
+        CheckResult result = CheckConfigUtil.checkAllExists(pluginConfig, NODE_URLS.key(), DATABASE.key(), TABLE.key(), USERNAME.key(), PASSWORD.key());
```
Comment (EricJoy2048) [inline seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/sink/StarRocksSink.java:56]: From the document I found `TABLE` is an option options?

Comment (TaoZex) [reply seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/sink/StarRocksSink.java:56]: After confirmation, I found that table was a required attribute and I have changed it. Thank you for your advice.

**Review by EricJoy2048 — APPROVED** [review]

**Comment by TaoZex** [comment]
@Hisoka-X PTAL

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**